csrc/scheduler/pointwise.cpp

```diff
@@ -295,10 +295,17 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
                 largest_out, true, true));
       });
 
+  int64_t max_dtype_size_for_vectorization = 1;
+  for (auto inp : vectorizable_inputs_outputs_entry.get()) {
```

Comment:
This seems to be an obvious mistake. It seems the transpose scheduler is fine, but the reduction scheduler is not. Can you also fix it as well?

Reply:
I don't think reduction scheduler shares the same problem. I think it's only the naming that's confusing.

I'm cleaning it up in #4141